site/src/docs/components/cookie-consent/code.mdx

```diff
@@ -0,0 +1,119 @@
+    optionalCookies: {
+      cookies: [
+          {
+            commonGroup: 'statistics',
+            commonCookie: 'matomo',
+          },
+        ],
+    },
+    language: {
+      onLanguageChange,
+    },
+    focusTargetSelector: '#focused-element-after-cookie-consent-closed',
+    onAllConsentsGiven: (consents) => {
+      // called when consents are saved
+      // handle changes like:
+      if (!consents.matomo) {
+        // stop matomo tracking
+      }
+    }
+  };
+
+  const Application = () => {
+    return (
+      <div>
+        <h1 id="focused-element-after-cookie-consent-closed" tabIndex={0}>
+          Simple cookie consent example
+        </h1>
+      </div>
+    );
+  };
+
+  return (
+    <>
+      <CookieModal contentSource={contentSource} />
+      <Application />
+    </>
+  );
+};
+```
+
+<Link href="/storybook/react/?path=/story/components-cookieconsent--modal-version">View working modal demo in Storybook</Link>
+
+### Rendered on a page
+
+```jsx
+import { CookiePage } from 'hds-react';
+
+export const PageVersion = () => {
+  const contentSource = {
+    siteName: 'Test site',
+    currentLanguage: 'en',
+    optionalCookies: {
+      cookies: [
+        {
+          commonGroup: 'statistics',
+          commonCookie: 'matomo',
+        },
+      ],
+    },
+    onAllConsentsGiven: (consents) => {
+      // called when consents are saved
+      // handle changes like:
+      if (!consents.matomo) {
+        // stop matomo tracking
+      }
+    }
+  };
+
+  return (
+    <main>
+      <CookiePage contentSource={contentSource} />
+    </main>
+  );
+};
+```
+
+<Link href="/storybook/react/?path=/story/components-cookieconsent--page-version">View working page example in Storybook</Link>
+
+### Packages
+| Package           | Included                      | Storybook link                                                                                                                      | Source link   |
+| ----------------- | ----------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| **HDS React**     | <IconCheckCircleFill /> Yes   | <Link size="M" external href="/storybook/react/?path=/story/components-cookieconsent--modal-version">View in Storybook</Link>  | <Link size="M" external href="https://github.com/City-of-Helsinki/helsinki-design-system/blob/master/packages/react/src/components/cookieconsent">View source</Link> |
+| **HDS Core**      | <IconCrossCircle /> No        | - | - |
+
+### Properties
+Note! You can find the full list of properties in the <Link size="M" href="/storybook/react/?path=/story/components-cookieconsent--default" external>React Storybook.</Link>
+
+Also, note that this component is an input. All features supported by the HDS TextInput are also supported by this component. See [TextInput documentation page](/components/text-input) for more information.
```

Comment:
This refers to TextInput component. It has probably been added as a base here, and we just have not set the actual properties here yet. Should we list the properties here, or add a link to the API tab?

Reply:
Fixed